src/extensions/default/HealthData/HealthDataManager.js

```diff
@@ -144,6 +146,9 @@ define(function (require, exports, module) {
                 
                 sendHealthDataToServer()
                     .done(function () {
+                        //We have already sent the health data, so can clear ll health data
+                        //Logged till now
```

Comment:
A space after `//` would be nice here and `ll` in `clear ll health` seems to be a typo.